tests/dataconverter/test_helpers.py

```diff
@@ -512,22 +512,11 @@ def fixture_filled_test_data(template, tmp_path):
             alter_dict(
                 TEMPLATE,
                 "/ENTRY[my_entry]/NXODD_name[nxodd_name]/char_value",
-                np.array(["1", "2", "3"], dtype=np.bytes_),
+                np.char.array(["1", "2", "3"]),
```

Comment:
Elements here are actually str, this is not a chararray before they were in np.character. In any case, this will be parsed element-wise so we don't need chararray. I would keep the old version here.